### ISSUE TITLE
Calendar Error

### ISSUE THREAD

**Issue by NCAvian** [makeuseofcode/GUI-Calendar-Using-Python#1]
I'm getting an error on line 9, in printCalendar:

  File "C:\Users\Bob\Desktop\Python\cal_gui.py", line 9, in printCalendar
    output_calendar = calendar.month(year, month)
AttributeError: module 'calendar' has no attribute 'month'

calendar package was installed with Anaconda. Python is version 3.9.13

When I import calendar, I see 3 functions (close, printCalendar, and reset). Month is not listed in the data fields.
Help shows the TclVersion = 8.6

**Comment by Rosilius** [comment]
Yep I have the same problem here. Did you manage to fix the issue?
